src/main/java/serviceObjects/ForcedAnswer.java

```diff
@@ -0,0 +1,33 @@
+package serviceObjects;
```

Comment:
The package should be named in lower case as convention:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html